src/command_modules/azure-cli-batchai/azure/cli/command_modules/batchai/custom.py

```diff
@@ -0,0 +1,390 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from __future__ import print_function
+
+import json
+import os
+import signal
+import time
+import requests
+from azure.cli.core.keys import is_valid_ssh_rsa_public_key
+
+from azure.cli.core.util import CLIError
+import azure.mgmt.batchai.models as models
+
+from msrest.serialization import Deserializer
+
+# Environment variables for specifying azure storage account and key. We want the user to make explicit
+# decision about which storage account to use instead of using his default account specified via AZURE_STORAGE_ACCOUNT
+# and AZURE_STORAGE_KEY.
+AZURE_BATCHAI_STORAGE_ACCOUNT = 'AZURE_BATCHAI_STORAGE_ACCOUNT'
+AZURE_BATCHAI_STORAGE_KEY = 'AZURE_BATCHAI_STORAGE_KEY'
+
+# Supported images.
+SUPPORTED_IMAGES = {
+    "ubuntults": models.ImageReference(
+        publisher='Canonical',
+        offer='UbuntuServer',
+        sku='16.04-LTS'
+    ),
+    "ubuntudsvm": models.ImageReference(
+        publisher='microsoft-ads',
+        offer='linux-data-science-vm-ubuntu',
+        sku='linuxdsvmubuntu'
+    )
+}
+
+
+def _get_deserializer():
+    client_models = {k: v for k, v in models.__dict__.items() if isinstance(v, type)}
+    return Deserializer(client_models)
+
+
+def get_environment_variable_or_die(name):
```

Comment:
Prefer to use the CLI config system for this so all config values pass through the same system.